ATTRIBUTIONS-Python.md

```
@@ -41287,4 +41287,38 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+- License: `NVIDIA Proprietary Software`
+
+NVIDIA Software License Agreement.
+See https://docs.nvidia.com/cuda/eula/ for full terms.
+
+## python-discovery (1.2.1)
+
+- PyPI: https://pypi.org/project/python-discovery/1.2.1/
+- `repository`: https://github.com/tox-dev/python-discovery
+- License: `MIT`
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)